src/Angor/Client/Pages/Release.razor

```diff
@@ -39,16 +39,14 @@
                 <span class="header-icon-wrapper">
                     <Icon IconName="signature" Width="32" Height="32" />
                 </span>
-                <h5 class="header-text">Claim released funds without a penalty</h5>
+                <h5 class="header-text">Claim funds free</h5>
```

Comment:
no no "free" is not a good word here, keep the old text even that it is long

Reply:
Claim funds without penalty